src/routes/cache-hooks/cache-hooks.service.ts

```diff
@@ -275,14 +287,62 @@ export class CacheHooksService {
             safeAddress: event.address,
           }),
         );
+        this._logMessageEvent(event);
         break;
       case EventType.CHAIN_UPDATE:
         promises.push(this.chainsRepository.clearChain(event.chainId));
+        this._logEvent(event);
         break;
       case EventType.SAFE_APPS_UPDATE:
         promises.push(this.safeAppsRepository.clearSafeApps(event.chainId));
+        this._logEvent(event);
         break;
     }
     return Promise.all(promises);
   }
+
+  private _logSafeTxEvent(
+    event: ExecutedTransaction | NewConfirmation | PendingTransaction,
+  ): void {
+    this.loggingService.info({
```

Comment:
I would move this to `debug` level given the scope of this change.

Reply:
Adding these lines to `debug` level, and enabling that level, would generate a lot of logging, and I think this change could be permanent so I'm more leaning towards using `info` if you agree 🙂